pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -108,6 +108,9 @@ type ClusterSpec struct {
 	// Indicates user intent when deleting a cluster; blocks orchestration and should not be set if cluster
 	// deletion is not imminent.
 	CleanupPolicy CleanupPolicySpec `json:"cleanupPolicy,omitempty"`
+
+	// The value of the `root` label in the CRUSH tree. Defaults to `default`.
+	CRUSHRoot string `json:"crushRoot,omitempty"`
```

Comment:
No need for a new setting defined here if it is part of the `storage.config` section.